docs/developers/contributing.md

```diff
@@ -23,12 +23,10 @@ Set the `upstream` remote to the base `napari` repository:
 git remote add upstream https://github.com/napari/napari.git
 ```
 
-Napari can be installed on most macOS, Linux, and Windows systems with Python 3.8, 3.9, and 3.10 using pip.  First, 
-create a clean virtual environment:
-```python
-conda create -y -n napari-env -c conda-forge python=3.9
-conda activate napari-env
-```
+Napari can be installed on most macOS, Linux, and Windows systems with 
+[![Supported Python versions](https://img.shields.io/pypi/pyversions/napari.svg)](https://python.org) by
+following the directions on the 
+[instructions page](https://napari.org/tutorials/fundamentals/installation.html#install-as-python-package-recommended).
```

Comment:
Linked to installation instructions rather than repeating here.

Reply:
```suggestion
[instructions page](https://napari.org/stable/tutorials/fundamentals/installation.html).
```

Reply:
Ideally, we should use internal links so that you don't get redirected to a different version of the page. So, 
```suggestion
[instructions page](../tutorials/fundamentals/installation.md#install-as-python-package-recommended).
```